psi4/driver/procrouting/empirical_dispersion.py

```diff
@@ -42,6 +42,7 @@
                                           ('dftd3', ['d2', 'd3zero', 'd3bj', 'd3mzero', 'd3mbj']),
                                           ('nl', ['nl']),
                                           ('mp2d', ['dmp2']),
+                                          ("dftd4", ["d4bjeeqatm"]),
```

Comment:
Double quotes here, but not above. Also on line 60 and throughout this file.

Reply:
yes, since black will clobber all my single quotes with double quotes someday, I've started to preempt it with doubles.